content/src/controller/Controller.ts

```diff
@@ -810,6 +822,12 @@ export enum DeploymentField {
   AUDIT_INFO = 'auditInfo'
 }
 
+export type PointerChangesField = 'authChain'
+
+export const pointerChangesFields: Record<string, PointerChangesField> = {
+  AUTH_CHAIN: 'authChain'
+}
```

Comment:
Is it really necessary to make the authChain optional for the responses? it adds branches and multiple paths to test and maintain, probably adding the value always is more straight forward and cheaper to maintain

Reply:
That was something that we weren't sure about. Let's always show it